Jenkinsfile

```diff
@@ -21,59 +21,63 @@ def secrets = [
   ]
 ]
 
-String dockerCompose = 'docker-compose -f integration-tests/docker-compose.yml -f docker-compose.yml --project-directory .'
```

Comment:
I decided to use `docker-compose.override.yml` which works great by default if there is one override (which is the case here). Thanks to this we can use all `docker-compose` files directly without any extra hassle.

Reply:
To explain the issue as @timja-kainos noted the problem was with `--project-directory`. @rlewan used that to enforce using current directory as project directory so that all test outcomes land in single place. Unfortunately that did not work well with mounting relative host volumes.

Generally docker-compose does not work well with many docker-compose.yml files which are scattered across many directories. Simples is to keep all such files in one place.